src/lib/features/client-feature-toggles/client-feature-toggle-store.ts

```diff
@@ -60,6 +70,8 @@ export default class FeatureToggleClientStore
                 action,
             });
         this.flagResolver = flagResolver;
+        const isTest = process.env.NODE_ENV === 'test';
+        this.isOss = !isEnterprise && ui.environment !== 'pro' && !isTest;
```

Comment:
Instead of having this logic in the constructor, can we do it outside and get this value injected? That might simplify testing

Reply:
yeah. let's do that. That way I can reuse it all the stores. Good feedback :hand: